dev/src/index.ts

```diff
@@ -226,7 +227,7 @@ export class Firestore {
    * The serializer to use for the Protobuf transformation.
    * @private
    */
-  private _serializer: Serializer|null = null;
+  _serializer: Serializer|null = null;
```

Comment:
```suggestion
  _serializer: Serializer | null = null;
```

Reply:
I'll try this but I bet clang format is going to change this back.

Reply:
(Yeah, it's back to what it was before)